04-spatial-operations.Rmd

```diff
@@ -41,53 +46,52 @@ It is important to note that spatial operations that use two spatial objects rel
 readers can see this default value of `op` set in the first line of the function call by entering its long-form name into the console: 
-`` sf:::`[.sf` ``
+`` sf:::`[.sf` `` 
+The `?sf` help page documents this also.
 ]
-
-There are 9 well-defined operations that can be used for spatial subsetting, covered in section \@ref(topological-relations).
+There are nine well-defined operations that can be used for spatial subsetting, covered in section \@ref(topological-relations).
 This may seem daunting but the good news is that you do not have to learn all of them separately:
 after you understand how to spatially subset objects that *intersect* another (via `st_intersects()`) it is easy to subset based on other types of spatial operation such as `st_touches()`, `st_crosses()` and `st_within()`.
 For this reason now we focus only on one of the spatial subsetting operations.
 We use `st_intersects()` instead of any of the others not only because it the default when subsetting with `[`,
 but also `st_intersects()` is useful as a 'catch all' that identifies all types of spatial relations.
 
 In general terms, spatial subsetting is simply the spatial equivalent of *attribute subsetting*.
-However, to do spatial subsetting *two spatial objects are needed* the spatial relation between which is to be established.
 As with attribute subsetting, spatial subsetting is a *binary operation*: an object is either selected or not.
-As in section \@ref(vector-attribute-subsetting), we start with base methods before describing how to do it in the **tidyverse**.
+Following the structure of section \@ref(vector-attribute-subsetting), we start with base methods before describing how to do it in the **tidyverse**.
 <!-- todo: link to non-binary links, e.g. area-weighted spatial interpolation -->
 
 <!-- ### Spatial subsetting in base R -->
 
-Attribute subsetting in base R is done with the `[` operator and passing into the square brackets a vector of class `integer` (whole numbers) or `logical` (a vector of `TRUE`s and `FALSE`s).
-This means `world[1:6, ]` subsets the first 6 countries of the world and that `world[world$area_km2 < 10000, ]` returns the subset of countries that have a small surface area.
-For this chapter we will use countries in Africa, which can be generated using this method as
-follows:^[Recall
+To introduce spatial vector subsetting operations we will use countries in Africa.
+We will apply attribute subsetting to the `world` dataset (see previous chapter):^[Recall
 that we can also subset simple features using the `filter()` function, e.g. with `filter(world, continent == "Africa")`]
 
 ```{r}
 africa_wgs = world[world$continent == "Africa", ]
 ```
 
-To further set-up the input data, we will reproject the data to the coordinate reference system (CRS) 32630 (it's EPSG code, explained in Chapter 6):
+To further set up the input data, we will reproject the data to the coordinate reference system (CRS) 32630, its EPSG code (explained in Chapter 6):
```

Comment:
I think setup is better - we should be consistent so worth searching for all instances. Set up sounds strange and stilted to me. They use setup in R4DS so let's go with that: http://r4ds.had.co.nz/introduction.html

Reply:
Ok, perfect.